File: windows/LanAudioBridge.Desktop/Core/Interfaces/ILogger.cs
using System;

namespace LanAudioBridge.Core;

/// <summary>
/// ILogger — 统一日志接口
/// </summary>
public interface ILogger
{
    void Debug(string tag, string msg);
    void Info(string tag, string msg);
    void Warn(string tag, string msg);
    void Error(string tag, string msg);
    void Error(string tag, string msg, Exception ex);
}

File: android/app/src/main/java/com/lanbridge/app/core/interfaces/ILogger.kt
package com.lanbridge.app.core.interfaces

interface ILogger {
    fun debug(tag: String, msg: String)
    fun info(tag: String, msg: String)
    fun warn(tag: String, msg: String)
    fun error(tag: String, msg: String)
    fun error(tag: String, msg: String, ex: Exception)
}
